shimmingtoolbox/optimizer/lsq_optimizer.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+import numpy as np
+import scipy.optimize as opt
+
+from shimmingtoolbox.optimizer.basic_optimizer import Optimizer
+
+
+class LSQ_Optimizer(Optimizer):
+
+    def _residuals(self, coef, unshimmed_vec, coil_mat):
+        """
+        Objective function to minimize
+        Args:
+            coef (numpy.ndarray): 1D array of channel coefficients
+            unshimmed_vec (numpy.ndarray): 1D flattened array (point) of the masked unshimmed map
+            coils_mat (numpy.ndarray): 2D flattened array (point, channel) of masked coils (axis 0 must align with unshimemd_vec)
```

Comment:
```suggestion
            coils_mat (numpy.ndarray): 2D flattened array (point, channel) of masked coils (axis 0 must align with unshimmed_vec)
```

Also line is too long for PEP8